.npmrc

```diff
@@ -1,14 +0,0 @@
-# Enforce Node and pnpm versions as specified in `package.json`
-engine-strict=true
-
-# Make installation of peer dependencies more predictible
-auto-install-peers=false
-strict-peer-dependencies=true
-
-# Save exact dependency versions in `package.json`
-save-prefix=""
-
-# Branch that must be checked out when publishing
-publish-branch=main
```

Comment:
Thought this was an npm config option but actually it's a CLI flag of [`pnpm publish`](https://pnpm.io/cli/publish#--publish-branch-branch), which already defaults to `master` or `main`. It can be set as an [option](https://pnpm.io/cli/publish#configuration) but we clearly no longer need to.

The remaining options move to `pnpm-workspace.yaml`, which means we no longer need an `.npmrc` file at all.